Makefile

```diff
@@ -317,7 +317,12 @@ test-unit: buildenv-dirs "$(KUSTOMIZE)"
 # small unit test to verify the behavior of an example kustomization for manual install
 .PHONY: test-kustomization
 test-kustomization:
-	$(MAKE) build-manifests STAGING_DIR=$(OUTPUT_DIR)/testing REGISTRY=gcr.io/cs-test IMAGE_TAG=placeholder
+	$(MAKE) build-manifests \
```

Comment:
I'd consider making this a separate change in case of revert/cherrypick, but no biggie

Reply:
https://github.com/GoogleContainerTools/kpt-config-sync/pull/1471